src/io/show/BalancedGrowthCommunityModel.jl

```diff
@@ -0,0 +1,13 @@
+function Base.show(io::Base.IO, ::MIME"text/plain", cm::CommunityMember)
+    println(
+        io,
+        "A $(typeof(cm.model)) community member with $(n_reactions(cm.model)) reactions, $(n_metabolites(cm.model)) metabolites, and abundance $(cm.abundance*100)%.",
+    )
+end
+
+function Base.show(io::Base.IO, ::MIME"text/plain", cm::BalancedGrowthCommunityModel)
+    println(
+        io,
+        "A balanced growth community model comprised of $(length(cm.members)) underlying models.",
+    )
+end
```

Comment:
this isn't very useful, I guess the generic show will suffice for now. I'll eventually make a bigger PR with `MultiModelWrapper` that fixes this forever

Reply:
(as in, pls remove the custom `show`s here)